public/src/actions/calls.jsx

```diff
@@ -118,5 +119,27 @@ export function updateCallAttempt(userId, campaignId, callId, outcome, notes = n
   .then(() => {
     dispatch(assignToCall(userId, campaignId));
   })
-  .catch();
+  .catch(err => err);
+}
+
+export function initateVolunteerTwilioCon(userId, campaignId, volunteerCallActive) {
+  if (volunteerCallActive) {
```

Comment:
So the issue here is that in some situations the client may end up with a stale idea of a volunteer's call state. It would be prudent to actually fetch a user's call state by GET on /users/:id and inspecting the call SID key in the response object.

Same comment for the method below. What do you think?